packages/xstate-test/test/plans.test.ts

```diff
@@ -74,4 +98,51 @@ describe('testModel.testPlans(...)', () => {
       ]
     `);
   });
+
+  describe('When the machine only has one path', () => {
+    it('Should only follow that path', () => {
+      const machine = createTestMachine({
+        initial: 'a',
+        states: {
+          a: {
+            on: {
+              EVENT: 'b'
+            }
+          },
+          b: {
+            on: {
+              EVENT: 'c'
+            }
+          },
+          c: {}
+        }
+      });
+
+      const model = createTestModel(machine);
+
+      const plans = model.getPlans();
+
+      expect(plans).toHaveLength(1);
+    });
+  });
```

Comment:
Can you also add a test for multiple transitions between states (e.g. `on: { ANOTHER_EVENT: 'b' }`) to ensure that there exists a path with those events?

This will pass of course, just a good sanity check to have